deploy/grafana-dashboard/runner_insights_row.py

```diff
@@ -0,0 +1,66 @@
+from grafanalib.core import RowPanel, GridPos, Histogram, TimeSeries
+from grafanalib.influxdb import InfluxDBTarget
+
+from util import read_query
+
+execution_duration_extra_json = {
+    "fieldConfig": {
+        "defaults": {
+            "unit": "ns"
+        }
+    }
+}
+execution_duration = Histogram(
+    title="Execution duration",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("execution-duration-hist"))],
+    gridPos=GridPos(h=8, w=24, x=0, y=2),
+    bucketSize=100000000,
+    colorMode="palette-classic",
+    extraJson=execution_duration_extra_json
+)
+
+executions_per_runner = Histogram(
+    title="Executions per runner",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("executions-per-runner-hist"))],
+    gridPos=GridPos(h=10, w=11, x=0, y=10),
+    bucketSize=1,
+    colorMode="palette-classic",
+)
+
+executions_per_minute = TimeSeries(
+    title="Executions per minute",
```

Comment:
I was wondering whether we should allow more fine granular data points here (similar to the current dashboard version) to account for smaller differences.

Reply:
I have adjusted this dashboard to use the dynamic value again.